### ISSUE TITLE
Elements not showing correctly

### ISSUE THREAD

**Issue by paulsybrandy** [KillerBOSS2019/TP-Speed-Test-Plugin#9]
![Screenshot_2024-06-13-17-42-41-14_a7b220fb958a21ad0385b6f6d25ca1e3](https://github.com/KillerBOSS2019/TP-Speed-Test-Plugin/assets/83975391/dece282d-6a96-4d7c-9e84-83bcc6bf2a5f)

As you can see from the screenshot, it looks as if the code within the page (and buttons) are not reading correctly. I redownloaded the plugin and page and imported them again but same issue. Is it something I am doing incorrectly?

**Comment by MaximusLeBastard** [comment]
Same problem here except, i managed to get the timezone right with timezone plugin through discord. But all the rest are like shown here except the cpu, gpu and ram they are also messed up..

**Comment by paulsybrandy** [comment]
> Same problem here except, i managed to get the timezone right with timezone plugin through discord. But all the rest are like shown here except the cpu, gpu and ram they are also messed up..

And unfortunately, there hasn't been any replies so I am not too hopeful for a solution :(

**Comment by MaximusLeBastard** [comment]
> > Same problem here except, i managed to get the timezone right with timezone plugin through discord. But all the rest are like shown here except the cpu, gpu and ram they are also messed up..
> 
> And unfortunately, there hasn't been any replies so I am not too hopeful for a solution :(

Yeah, I was wondering if I was doing something wrong to get it to work. I tried some other plugins that I thought I needed, but it seems nothing worked except the timezone plugin. So, we must do something else to get it to work I guess.
